subscriber_topic_1: /line_detector/segment_list

publisher_topic_1: /birds_eye/segment_list

#homography: [[-6.24454148e-01, -1.50764192e+00, 1.03724891e+03], [-7.66053887e-15, -1.96506550e+00, 9.03930131e+02], [-9.86623977e-18, -2.37445415e-03, 1.00000000e+00]]
#homography: [[ -1.08049113e+00 , -2.53206003e+00  , 1.33151432e+03],[ -5.77315973e-15 , -3.14324693e+00  , 9.11541610e+02] , [ -6.93889390e-18 , -3.95634379e-03  , 1.00000000e+00]]

homography: [[ -3.15315315e-01,  -1.63363363e+00,   4.24054054e+02], [ -3.71230824e-16,  -3.14114114e+00,   7.98918919e+02], [  2.04643160e-18,  -5.10510511e-03,   1.00000000e+00]]



segment_warper_sub_1: /line_detector/segment_list

subs_queue_size: 1
pubs_queue_size: 1
buff_size: 52428800
default_sobel_kernel_size: 7

img_size: [376, 672]

# Values for Region of Interest (ROI) Cutter
left_bottom: [20, 376]
right_bottom: [652, 376]
apex1: [310, 140]
apex2: [350, 140]
inner_left_bottom: [40, 376]
inner_right_bottom: [336, 376]
inner_apex1: [340, 480]
inner_apex2: [320, 480]

color_config:
  dilatation_kernel_size: 3
  canny_threshold: [80, 200]
  hough_threshold: 20
  hough_min_line_length: 3
  hough_max_line_gap: 1
  sobel_threshold: 40

  hsv_white1:  [0,0,150]
  hsv_white2:  [180,50,255]
  hls_white1:  [0, 200, 0]
  hls_white2:  [180, 255, 255]
  hsv_yellow1: [25,120,90]
  hsv_yellow2: [45,255,255]
  hls_yellow1: [15, 38, 115]
  hls_yellow2: [35, 204, 255]
  hsv_red1:    [0,140,100]
  hsv_red2:    [15,255,255]
  hsv_red3:    [165,140,100]
  hsv_red4:    [180,255,255]
